charts/tidb-cluster/values.yaml

```diff
@@ -194,10 +195,17 @@ fullbackup:
   # https://kubernetes.io/docs/tasks/job/automated-tasks-with-cron-jobs/#starting-deadline
   startingDeadlineSeconds: 3600
   # https://github.com/maxbube/mydumper/blob/master/docs/mydumper_usage.rst#options
-  options: "--rows=10000"
+  options: "--chunk-filesize=100"
   # user must set tidb cluster user and password
   user: root
```

Comment:
Can you change this from `root` to `backup`? Right now it seems like using backup exposes the credentials? Can we move the password into the secret system?